src/printer.ts

```diff
@@ -550,6 +611,23 @@ export class PugPrinter {
 		return this.quoteString(val);
 	}
 
+	private formatSvelteInterpolation(val: string): string {
```

Comment:
This function is a copy of `formatAngularInterpolation` but with checking for single curly brackets.
Maybe we can merge them and make it more dynamic. Or maybe we need to find out if it's useful for other cases then only `svelte` :thinking:

Reply:
IMO you should merge these two methods into a single one and use the existing methods to call that method with the right parameters.

- `private formatAngularInterpolation(val: string): string {`
- `private formatSvelteInterpolation(val: string): string {`
- `private formatFrameworkInterpolation(val: string, quotes: string[]): string {` (not sure about the name 😉 )